homeassistant/components/renson_endura_delta/const.py

```diff
@@ -0,0 +1,218 @@
+    key="HUMIDITY_FIELD",
+    name="Relative humidity",
+    device_class=DEVICE_CLASS_HUMIDITY,
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement="%",
+)
+
+MANUAL_LEVEL_DESC = SensorEntityDescription(
+    key="MANUAL_LEVEL_FIELD",
+    name="Manual level",
+    state_class=STATE_CLASS_MEASUREMENT,
+)
+
+TIME_AND_DATE_DESC = SensorEntityDescription(
+    key="MANUAL_LEVEL_FIELD",
+    name="System time",
+    device_class=DEVICE_CLASS_TIMESTAMP,
+    state_class=STATE_CLASS_MEASUREMENT,
+)
+
+BREEZE_TEMPERATURE_DESC = SensorEntityDescription(
+    key="BREEZE_TEMPERATURE_FIELD",
+    name="Breeze temperature",
+    device_class=TEMP_CELSIUS,
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement=TEMP_CELSIUS,
+)
+
+BREEZE_LEVEL_DESC = SensorEntityDescription(
+    key="BREEZE_LEVEL_FIELD",
+    name="Breeze level",
+    state_class=STATE_CLASS_MEASUREMENT,
+)
+
+DAYTIME_DESC = SensorEntityDescription(
+    key="DAYTIME_FIELD",
+    name="Start day time",
+    state_class=STATE_CLASS_MEASUREMENT,
+)
+
+
+NIGHTTIME_DESC = SensorEntityDescription(
+    key="NIGHTTIME_FIELD",
+    name="Start night time",
+    state_class=STATE_CLASS_MEASUREMENT,
+)
+
+DAY_POLLUTION_DESC = SensorEntityDescription(
+    key="DAY_POLLUTION_FIELD",
+    name="Day pollution level",
+    state_class=STATE_CLASS_MEASUREMENT,
+)
+
+NIGHT_POLLUTION_DESC = SensorEntityDescription(
+    key="NIGHT_POLLUTION_FIELD",
+    name="Night pollution level",
+    state_class=STATE_CLASS_MEASUREMENT,
+)
+
+CO2_THRESHOLD_DESC = SensorEntityDescription(
+    key="CO2_THRESHOLD_FIELD",
+    name="CO2 threshold",
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement="ppm",
+)
+
+CO2_HYSTERESIS_DESC = SensorEntityDescription(
+    key="CO2_HYSTERESIS_FIELD",
+    name="CO2 hysteresis",
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement="ppm",
+)
+
+BYPASS_TEMPERATURE_DESC = SensorEntityDescription(
+    key="BYPASS_TEMPERATURE_FIELD",
+    name="Bypass activation temperature",
+    device_class="temperature",
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement=TEMP_CELSIUS,
+)
+
+BYPASS_LEVEL_DESC = SensorEntityDescription(
+    key="BYPASS_LEVEL_FIELD",
+    name="Bypass level",
+    device_class="power_factor",
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement="%",
+)
+
+FILTER_PRESET_DESC = SensorEntityDescription(
+    key="FILTER_PRESET_FIELD",
+    name="Filter preset time",
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement="days",
+)
+
+PREHEATER_DESC = BinarySensorEntityDescription(
+    key="PREHEATER_FIELD",
+    name="Preheater enabled",
+)
+
+HUMIDITY_CONTROL_DESC = BinarySensorEntityDescription(
+    key="HUMIDITY_CONTROL_FIELD",
+    name="Humidity control enabled",
+)
+
+AIR_QUALITY_CONTROL_DESC = BinarySensorEntityDescription(
+    key="AIR_QUALITY_CONTROL_FIELD",
+    name="Air quality control enabled",
```

Comment:
You can leave out the word `enabled` of all of these. The state is already `on` or `off`.